iron-iconset-svg.html

```diff
@@ -63,14 +63,6 @@
       },
 
       /**
-       * Array of fully-qualitifed icon names in the iconset.
-       */
-      iconNames: {
```

Comment:
I have no magic sauce. It's a _breaking change_, but I can't avoid it. We can avoid a major version number only by committing a (pious?) fraud.